bin/sync_slurm_acct.py

```diff
@@ -141,21 +141,28 @@ def main():
         sacctmgr_commands += slurm_vo_accounts(account_page_vos, slurm_account_info, clusters, host_institute)
 
         # process VO members
-        [job_cancel_commands, user_commands] = slurm_user_accounts(
+        (job_cancel_commands, user_commands, association_remove_commands) = slurm_user_accounts(
             account_page_members,
             active_accounts,
             slurm_user_info,
             clusters,
             opts.options.dry_run
         )
 
+        # Adding users takes priority
+        sacctmgr_commands += user_commands
+
         # safety to avoid emptying the cluster due to some error upstream
-        if not opts.options.force and len(job_cancel_commands > 5):
-            logging.error("Aborting, would otherwise cancel jobs for %d users", len(job_cancel_commands))
-            raise SyncSanityError("Would cancel jobs for %d users" % len(job_cancel_commands))
+        if not opts.options.force and len(job_cancel_commands) > 5:
+            logging.warning("Will not add commands to cancel jobs for %d users", len(job_cancel_commands))
```

Comment:
fix message to state that this is number of cancel jobs, not user

Reply:
how so?

Reply:
it's a defaultdict, so `len` is the number of keys/values, right?